test/functional/driver/driver-e2e-specs.js

```diff
@@ -83,30 +83,6 @@ describe('XCUITestDriver', function () {
         simsAfter.should.equal(simsBefore);
       });
 
-      it('with udid: uses sim and shuts it down afterwards', async () => {
```

Comment:
@mykola-mokhnach isn't this testing the wrong thing in the 2nd-to-last validation? shouldn't `simsDuring` be equal to `simsBefore + 1`?